packages/fuels-test-helpers/src/lib.rs

```diff
@@ -142,7 +143,7 @@ pub async fn setup_test_client(
     messages: Vec<Message>,
     node_config: Option<Config>,
     chain_config: Option<ChainConfig>,
-) -> (FuelClient, SocketAddr, ConsensusParameters) {
+) -> Result<(FuelClient, SocketAddr, ConsensusParameters), Error> {
```

Comment:
When spawning a new node with 
```rust
let srv = FuelService::new_node(config).await.unwrap();
```
we could map and propagate the error now that we return a result.